src/components/NumberInput/NumberInput.js

```diff
@@ -105,6 +105,24 @@ export default class NumberInput extends Component {
       value: this.state.value,
     };
 
+    const buttonProps = {
+      type: 'button',
+      className: 'bx--number__control-btn',
+      disabled
+    };
+
+    const createButton = dir => (
```

Comment:
refactored this so its dry'er

Reply:
Do we need to do this if it's only at two call sites?

Reply:
1 < 2 so id say yes

Reply:
I'm less concerned about file size and more concerned about closure allocs. I'm definitely down if the change made things much easier, but since it's so few call sites the change seems unnecessary.